1001-2000/LIT1651Homily.xml

```diff
@@ -1,12 +1,17 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT1651Homily">
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:lang="gez" xml:id="t1">ድርሳን፡ ዘ፬እንስሳ፡ መንፈሳውያን፡ ወብርሃናውያን፡</title>
-            <title xml:lang="gez" corresp="#t1" type="normalized">Dǝrsān za-4 ʾǝnsǝsā manfasāwǝyān wa-bǝrhānāwǝyān</title>
-            <title xml:lang="en" xml:id="t2">Homily on the Four Living Creatures</title>
+           <title xml:lang="en" corresp="#t1">Homily on the Four Living Creatures by John Chrysostom</title>
```

Comment:
Is the first line the title you want to have in the article?

Reply:
Not really. Corrected.